crates/shim-client/README.md

```diff
@@ -27,8 +48,11 @@ let resp = task_client.connect(context, &req)?;
 
 ## Example
 
-Have a look on example [here](./examples/connect.rs).
+- [ttRPC shim client](./examples/ttrpc-client.rs)
+- [ttRPC shim server](./examples/ttrpc-server.rs)
+- [ttRPC client connect](./examples/connect.rs).
```

Comment:
nit: TTRPC similarly to GRPC?

Reply:
Pardon, didn't get your point here. Should I mention that GRPC is similar to ttRPC so the example may work for GRPC too?

Reply:
No, I meant like rename `ttRPC` to just `TTRPC`

Reply:
I'll go ahead and merge this one. It's not a blocker and easy to follow up.